src/neo/NeoSystem.cs

```diff
@@ -133,6 +133,7 @@ public static Block CreateGenesisBlock(ProtocolSettings settings) => new()
                 PrevHash = UInt256.Zero,
                 MerkleRoot = UInt256.Zero,
                 Timestamp = (new DateTime(2016, 7, 15, 15, 8, 21, DateTimeKind.Utc)).ToTimestampMS(),
+                Nonce = 2083236893, // nonce from the Bitcoin genesis block.
```

Comment:
Any meaning?

Reply:
> Any meaning?

nop, we dont really need any meaningful nonce here, it is just a placeholder, so i picked the nonce from the bitcoin genesis block to show our respect to Satoshi Nakamoto.